homeassistant/components/smtp/notify.py

```diff
@@ -182,7 +183,11 @@ def send_message(self, message="", **kwargs):
             msg = _build_text_msg(message)
 
         msg["Subject"] = subject
-        msg["To"] = ",".join(self.recipients)
+
+        recipients = kwargs.get(ATTR_TARGET)
```

Comment:
Don't we want this to be a member `recipients` in the `data` dict instead, or are there other integrations using `target` in this way?

Reply:
Yes, target is the generic field to specify recipients in a Notifications service call. See the doc here: https://www.home-assistant.io/integrations/notify/

This is very useful for sending one time password using email. For the current implementation, each user will have to create it's own SMTP setup with recipients hardcoded, then use that specific setting as the auth_mfa_modules.

With this change, the admin can setup a SMTP entry as the "server", with a default recipient. Then each user can utilize the GUI and use this SMTP server entry but specify their own target email address. Please not that "target" field is directly configurable via the MFA notify GUI, but "data" field is not.